src/common/components/RatingSection/RatingSection.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  ratingSectionTheme,
+  type RatingSectionVariants,
+} from "./RatingSection.theme";
+import { HeartIcon } from "@/common/components/CustomIcon";
+import { LockCtaOverlay } from "@/common/components/LockCtaOverlay/LockCtaOverlay";
+import { StatItem } from "@/common/components/StatItem";
+import { formatPercentage } from "@/common/functions/format";
+
+export type RatingItem = {
+  label: string;
+  rating: number;
+};
+
+export type RatingSectionProps = RatingSectionVariants & {
+  headingRatingItem: RatingItem;
+  ratingItems: RatingItem[];

```

Comment:
Could we just use the props of StatItem? So if the API changes for that component, we would be prompted to update this component too

Reply:
sure! updated at a0c73fe